umm-spec-lib/src/cmr/umm_spec/field_update.clj

```diff
@@ -6,11 +6,60 @@
   [cmr.umm-spec.date-util :as date-util]
   [cmr.umm-spec.umm-spec-core :as spec-core]))
 
+(def DataCenterURL
+  "Data center URL's content type."
+  "DataCenterURL")
+
+(def HomePage
+  "Data center URL's type."
+  "HOME PAGE")
+
 (defn field-update-functions
   "Partial update functions for handling specific update cases"
   [umm]
   {[:Instruments] (partial util/update-in-each umm [:Platforms])})
 
+(defn- get-home-page-url
+  "Get data center home page url from a list of related urls."
```

Comment:
The code is only choosing one home page URL. We should update the docstring to note that if there are more than one home page URLs we are arbitrarily choosing the first one.

Reply:
Done.